Update optimized-one-page-checkout.md (#850)
bigcommerce/dev-docs: docs/stencil-docs/customizing-checkout/optimized-one-page-checkout.md

<div class="otp" id="no-index">

### On this page
- [Working with the optimized checkout SCSS file](#working-with-the-optimized-checkout-scss-file)
- [Configuring the desktop viewport](#configuring-the-desktop-viewport)
- [Classes available for customization](#classes-available-for-customization)

> - [Providing customizable options in Store Design](#providing-customizable-options-in-store-design)

Reviewer: ```suggestion
- [Providing customizable options in Page Builder](#providing-customizable-options-in-page-builder)

```

- [Currency conversion options](#currency-conversion-options)

> - [Resources](#resources)

Reviewer: ```suggestion
- [Relater resources](#related-resources)

```

</div> 

## Working with the optimized checkout SCSS file

> Cornerstone includes a SCSS file that styles the Optimized Checkout page. In your local installation, this file is located at <span class="fn">cornerstone/assets/scss/optimized-checkout.scss</span>.

Reviewer: Should Optimized Checkout be lowercase to match the header?

Author: yes it should be lowercase, isn't it showing that way?

<div class="otp" id="no-index">

### On this page
- [Working with the optimized checkout SCSS file](#working-with-the-optimized-checkout-scss-file)
- [Configuring the desktop viewport](#configuring-the-desktop-viewport)
- [Classes available for customization](#classes-available-for-customization)
- [Providing customizable options in Page Builder](#providing-customizable-options-in-page-builder)
- [Currency conversion options](#currency-conversion-options)
- [Related resources](#related-resources)

</div> 

## Working with the optimized checkout SCSS file

Cornerstone includes a SCSS file that styles the Optimized Checkout page. In your local installation, this file is located at <span class="fn">`cornerstone/assets/scss/optimized-checkout.scss`</span>.

> You can access the most recent version of this file in the [Cornerstone repository](https://github.com/bigcommerce/cornerstone/blob/master/assets/scss/optimized-checkout.scss) (Github). All themes share the same stylesheet for the Optimized One-Page checkout, even those not based on Cornerstone, so this stylesheet applies universally across Stencil themes.

Reviewer 2: ```suggestion
You can access the most recent version of this file in the [Cornerstone repository](https://github.com/bigcommerce/cornerstone/blob/master/assets/scss/optimized-checkout.scss). All themes share the same stylesheet for the Optimized One-Page checkout, even those not based on Cornerstone, so this stylesheet applies universally across Stencil themes.

```

<div class="otp" id="no-index">

### On this page
- [Working with the optimized checkout SCSS file](#working-with-the-optimized-checkout-scss-file)
- [Configuring the desktop viewport](#configuring-the-desktop-viewport)
- [Classes available for customization](#classes-available-for-customization)
- [Providing customizable options in Page Builder](#providing-customizable-options-in-page-builder)

- [Currency conversion options](#currency-conversion-options)
- [Related resources](#related-resources)

</div> 

## Working with the optimized checkout SCSS file

> Cornerstone includes a SCSS file that styles the Optimized Checkout page. In your local installation, this file is located at <span class="fn">cornerstone/assets/scss/optimized-checkout.scss</span>.

Reviewer 2: All of the <span class="fn"></span> instances in the document can be replaced with backticks for consistency.

<!-- theme: error -->

### Customizing checkout restrictions
> When customizing the checkout page, you are free to change classes' contents; however, **do not** nest elements or change any class names. BigCommerce imposes these restrictions because each class here maps to multiple optimized checkout elements. Changing the structure or naming would break updates available in future optimized checkout enhancements.

</div>
</div>
</div>

## Configuring the desktop viewport

> To set up optimized checkout's responsive features within the <span class="fn">optimized-checkout.scss</span> file, you should configure the following breakpoint to define your preferred desktop viewport size:

Reviewer 2: ```suggestion
To set up optimized checkout's responsive features within the `optimized-checkout.scss` file, you should configure the following breakpoint to define your preferred desktop viewport size:

```

    }
```

## Classes available for customization

Below are the classes provided in Cornerstone's <span class="fn">optimized-checkout.scss</span>, with the corresponding page elements that they style. _These class names and mappings are subject to change, so please check the inline comments in the <span class="fn">optimized-checkout.scss</span> file that you downloaded with your current Cornerstone release._

> | **Class** | **Styles this Optimized One-Page Checkout element:** |

Reviewer: Should Optimized Checkout be all lowercase? Or should it be capitalized?

Author: In this case, I thought it should be capitalized because it is a name. The other occurrences were descriptions or adjectives.

    }
```

## Classes available for customization

> Below are the classes provided in Cornerstone's <span class="fn">`optimized-checkout.scss`</span>, with the corresponding page elements that they style. _These class names and mappings are subject to change, so please check the inline comments in the <span class="fn">`optimized-checkout.scss`</span> file that you downloaded with your current Cornerstone release._

Reviewer 2: ```suggestion
Below are the classes provided in Cornerstone's `optimized-checkout.scss`, with the corresponding page elements that they style.  These class names and mappings are subject to change, so please check the inline comments in the `optimized-checkout.scss` file that you downloaded with your current Cornerstone release.

```

| `.optimizedCheckout-form-label` | Styles form fields like `Email Address` – the field's text-label color. |
| `.optimizedCheckout-form-input` | Styles form fields like `Email Address` – the entry box's background and border colors. |

> ## Providing customizable options in Store Design

Reviewer: ```suggestion
## Providing customizable options in Page Builder

```

> You can determine which aspects of Optimized One-Page Checkout merchants will customize with the Store Design tool.

Reviewer: ```suggestion
You can determine which aspects of Optimized One-Page Checkout merchants will customize with the Page Builder tool.

```

> In Cornerstone's <span class="fn">optimized-checkout.scss</span>, each SCSS class and property available has a corresponding key/value pair in the <span class="fn">config.json</span> file, which can be used as the `id` value in schema.json when [creating customizable Store Design options](/stencil-docs/store-design/schema-json-metadata).

Reviewer: Updated the link since /stencil-docs/store-design/schema-json-metadata redirects to https://github.com/bigcommerce/cornerstone/blob/master/schema.json
```suggestion
In Cornerstone's <span class="fn">optimized-checkout.scss</span>, each SCSS class and property available has a corresponding key/value pair in the <span class="fn">config.json</span> file, which can be used as the `id` value in schema.json when [creating customizable Page Builder options](https://github.com/bigcommerce/cornerstone/blob/master/schema.json).

```

| `.optimizedCheckout-form-label` | Styles form fields like `Email Address` – the field's text-label color. |
| `.optimizedCheckout-form-input` | Styles form fields like `Email Address` – the entry box's background and border colors. |

## Providing customizable options in Page Builder

You can determine which aspects of Optimized One-Page Checkout merchants will customize with the Page Builder tool.

> In Cornerstone's <span class="fn">`optimized-checkout.scss`</span>, each SCSS class and property available has a corresponding key/value pair in the <span class="fn">`config.json`</span> file, which can be used as the `id` value in schema.json when [creating customizable Page Builder options](https://github.com/bigcommerce/cornerstone/blob/master/schema.json).

Reviewer 2: ```suggestion
In Cornerstone's `optimized-checkout.scss`, each SCSS class and property available has a corresponding key/value pair in the `config.json` file, which can be used as the `id` value in schema.json when [creating customizable Page Builder options](https://github.com/bigcommerce/cornerstone/blob/master/schema.json).

```
